standalone-metastore/metastore-server/src/main/sql/mysql/upgrade-3.2.0-to-4.0.0.mysql.sql

```diff
@@ -88,9 +88,9 @@ PREPARE stmt FROM @s;
 EXECUTE stmt;
 DEALLOCATE PREPARE stmt;
 RENAME TABLE NEXT_TXN_ID TO TXN_LOCK_TBL;
-ALTER TABLE TXN_LOCK_TBL RENAME COLUMN NTXN_NEXT TO TXN_LOCK;
+ALTER TABLE TXN_LOCK_TBL CHANGE NTXN_NEXT TXN_LOCK bigint;
```

Comment:
My guess is that this was caused by the difference between MySQL and MariaDB. If I remember correctly the test uses MariaDB. Could you please check manually with MySQL 5 and MySQL 8?
Thanks,
Peter

Reply:
In Mysql 8 both the syntax works fine but in MySql 5.7 Rename fails.